### DISCUSSION TITLE
ESP-CAM + DS18B20 Temperatur Sensor

### DISCUSSION THREAD

**Discussion by Sub-7** [jomjol/AI-on-the-edge-device#2240] [Q&A]
Gibt es eine Möglichkeit  DS18B20 Temperatur Sensor (oder sogar mehrere) parallel mit der Cam zu betreiben?
Wäre ganz toll.

![DS18B20](https://user-images.githubusercontent.com/30667941/227721993-30f4d91e-38dc-445b-8391-30f556673c53.jpg)

**Comment by caco3** [comment]
Nein, die Firmware unterstützt das so nicht.
Du müsstest einen der freien GPIOs verwenden und die Firmware selber erweitern.
